Terra/1000Genomes_GWAS_Template.md

```diff
@@ -55,15 +50,12 @@ Next, perform mixed models genetic association tests (run as a series of batch w
 # Part 1: Navigate the NHLBI BioData Catalyst ecosystem
 
 ## 1a. Link your Terra account to Gen3 using external services
-Before you're able to access genomic data from Gen3 in the Terra data table,  you need to link your Terra account to external services. Link your profile [by following these instructions](https://support.terra.bio/hc/en-us/articles/360038086332?flash_digest=2f492682b688b21da27c701af68656ac095d5803).
+Before you're able to access genomic data from Gen3 in the Terra data table, you need to link your Terra account to external services. If you have not already done so, link your profile [by following these instructions](https://support.terra.bio/hc/en-us/articles/360038086332?flash_digest=2f492682b688b21da27c701af68656ac095d5803).
```

Comment:
Allie at Broad mentioned to shorten URLs to ensure they remain correct (for example, if they change a title it could affect the URL). Here, all you need is "https://support.terra.bio/hc/en-us/articles/360038086332"